include/onnxruntime/core/optimizer/graph_transformer.h

```diff
@@ -5,7 +5,7 @@
 
 #include "core/common/common.h"
 #include "core/graph/graph_viewer.h"
-#include "core/graph/rewrite_rule.h"
+#include "core/optimizer/rewrite_rule.h"
```

Comment:
[Comment for all rewrite rule classes:]
Given we do the restructuring, I think it would be good to put all rewrite rules in a subfolder, like optimizer/rewrite_rules to easily identify them.

Reply:
Sounds good, we can do it later.